### DISCUSSION TITLE
Status Toggle Switches

### DISCUSSION THREAD

**Discussion by grandmastermarco** [MichelFR/MqDockerUp#134] [Ideas]
Hi. Great project.
On the readme Page, you state that it is possible to start stop and restart a container with mqdockerup. I dont understand how this is possible.
Maybe to make things easier, would it be a possibility to add a toggle switch to each container entity, to Show status and Also start stop the container easly directly from Lovelace?
Maybe even some Kind of update Button?

Thank you in advance!

**Comment by MichelFR** [comment]
I had to remove the functionality as it wasn’t functioning as I wanted it to be.
I’m gonna implement it soon.

im also gonna add a „button“ to update.
So that it would be possible to manually force an update. (Pull & recreate)
